openapi_python_client/templates/types.py

```diff
@@ -1,8 +1,11 @@
 """ Contains some shared types for properties """
-from typing import BinaryIO, Generic, MutableMapping, Optional, TextIO, Tuple, TypeVar, Union
+from typing import BinaryIO, Generic, MutableMapping, NewType, Optional, TextIO, Tuple, TypeVar, Union
 
 import attr
 
+Unset = NewType("Unset", object)
+UNSET: Unset = object()
```

Comment:
Shouldn't this be
```suggestion
Unset = NewType("Unset", object)
UNSET: Unset = Unset(object())
```

Reply:
I guess another alternative would be to keep it as `UNSET = object()` and use `Literal[Unset]` as the type? (Also doesn't seem like it matters)

Reply:
When I tried that previously, PyCharm complained:

```
'Literal' may be parameterized with literal ints, byte and unicode strings, bools, Enum values, None, other literal types, or type aliases to other literal types 
```